config/sara/README.md

```diff
@@ -1,38 +1,53 @@
 Note: Sub-jurisdictinos can add symptoms to groups, but should not be specifying their own groups.
+
 ### required
-The `required` field can either take the value `true` of `false`. Only symptoms with `required` set to `true` will be taken into consideration during symptomatic assessment calculation. Additionally, symptoms that have the `required` attribute set to `false` will be omitted from the voice and SMS assessment prompts.
+
+The `required` field can either take the value `true` or `false`. Only symptoms with `required` set to `true` will be taken into consideration during symptomatic assessment calculation. Additionally, symptoms that have the `required` attribute set to `false` will be omitted from the voice and SMS assessment prompts.
 
 ## Translations and Notes
+
 Each symptom supports the inclusion of a more descriptive sentence about what the symptom entails. This, as well as the translations for each symptom name are defined in a locale file under `config/locales`. When adding symptoms, you MUST include entries for these symptoms in each locale!
 
-### Example Use:
+## Example:
 
-In the configuration below, the USA jurisdiction will have 7 symptoms, these symptoms will apply to the
-USA jurisdiction as well as ALL of it's nested children, meaning that all jurisdictions all the way down
-to the county-level jurisdictions will inherit these symptoms. State 1 has specified it's own symptoms which
+In the configuration below, the USA jurisdiction will have 3 symptoms, these symptoms will apply to the
+USA jurisdiction as well as ALL of its nested children, meaning that all jurisdictions all the way down
+to the county-level jurisdictions will inherit these symptoms. State 1 has specified its own symptoms which
 will be added to the symptoms that it inherited from its parent jurisdiction, these symptoms will be applied
-to State 1, and the children of State1 (County 1  and County 2). In other words, a monitoree in State 1,
-County 1 or County 2 will be asked about 8 total symptoms as part of their assessments, whereas a monitoree in State 2 or County 3 would only be asked about 7 symptoms. Assessments which have 2 or more of the required group symptoms (Fever, Chills, Repeated Shaking with Chills, New Loss of Taste or Smell) will be considered symptomatic.
+to State 1, and the children of State 1 (County 1 and County 2). In other words, a monitoree in State 1,
+County 1 or County 2 will be asked about 5 total symptoms as part of their assessments, whereas a monitoree in State 2 or County 3 would only be asked about 3 symptoms. Assessments which have 2 or more of the required group symptoms (Fever) will be considered symptomatic.
```

Comment:
The last sentence confuses me. I think it should just be removed, unless I'm just being dumb (in which case let me know)
```suggestion
County 1 or County 2 will be asked about 5 total symptoms as part of their assessments, whereas a monitoree in State 2 or County 3 would only be asked about 3 symptoms.
```